### DISCUSSION TITLE
School project need help

### DISCUSSION THREAD

**Discussion by elpidoforo** [mcguirepr89/BirdNET-Pi#1228] [Q&A]
hello i am a teacher i want to create with my students a solar powered BirdNET-Pi  for our schools we are thinking to use raspberry pi zero wo2 because of its low power usage what hardware will i need to buy can you please help ?

**Comment by Nachtzuster** [comment]
There are no real recipies out there, I think, but you might find some inspiration here in the discussions and show and tell.
Also have a look here for inspiration:
https://github.com/Nachtzuster/BirdNET-Pi/discussions/40
